lib/lotus/router.rb

```diff
@@ -341,7 +341,7 @@ def defined?
     #    # It will map to Flowers::Index.new, which is the
     #    # Lotus::Controller convention.
     def get(path, options = {}, &blk)
-      @router.get(path, options, &blk)
+      @router.prefixed_namespace.get(path, options, &blk)
```

Comment:
@AlfonsoUceda This leaks `Lotus::Routing::HttpRouter` details to the higher level. That low level class should hide this detail. `Lotus::Router#get` should remain untouched, while `Lotus::Routing::HttpRouter#get` should handle this.